### PR TITLE
Add jboss-logging-processor to the BOM

### PR DESCRIPTION
**Pull request by dteleguin**
Required by the applications using `@MessageLogger`s from JBoss Logging.

### CODE REVIEW

**Comment by geoand** [comment]
Thanks @dteleguin! LGTM but I'll let the BOM master @gsmet have the final say :)

**Review by gsmet — APPROVED** [review]
Yes, we discussed it on Zulip.

Thanks for the PR, let's wait for CI.

**Pull request merged**
